guides/10_Video_Processor.md

```diff
@@ -0,0 +1,115 @@
+`DeviceController.chooseVideoInputDevice` accepts `VideoTransformDevice` to transform the `Device` given the `VideoFrameProcessor`s.
+
+A simplified workflow would be:
+
+1. Create an array of `VideoFrameProcessor`s.
+2. Create `VideoTransformDevice` with input `Device` and the array of `VideoFrameProcessor`s.
+3. Call `meetingSession.audioVideo.chooseVideoInputDevice` with the `VideoTransformDevice`.
+
+
+### Browser compatibility
+
+The APIs for video processing in Amazon Chime SDK for JavaScript works in Firefox, Chrome, Chromium-based browsers (including Electron) on desktop and Android operating systems. A full compatibility table is below. Currently, it does not support Safari on iOS devices. WebRTC Unified Plan is also required. Unified Plan is by default enabled in [MeetingSessionConfiguration.enableUnifiedPlanForChromiumBasedBrowsers](https://aws.github.io/amazon-chime-sdk-js/classes/meetingsessionconfiguration.html#enableunifiedplanforchromiumbasedbrowsers).
+
+
+|Browser                                                                |Minimum supported version  
+|---                                                                    |---                        
+|Firefox                                                                |76                        
+|Chromium-based browsers and environments, including Edge and Electron  |78                                             
+|Android Chrome                                                         |78       
+|Safari on MacOS                                                        |13.0                    
+|iOS Safari                                                             |Not supported             
+
+## Video Processing APIs and Usage
+
+### VideoTransformDevice
+`VideoTransformDevice` allows `VideoFrameProcessor`s to be applied to a `Device`.
+`DefaultVideoTransformDevice` is the provided implementation of `VideoTransformDevice`. It needs four parameters: (1) `Logger` (2) `Device` (3) `Array<VideoFrameProcessor>` (4) `DefaultBrowserBehavior`.
+
+```typescript
+class DefaultVideoTransformDevice {
+  constructor(
+    logger: Logger,
+    device: Device,
+    processors: VideoFrameProcessor[],
+    browserBehavior: DefaultBrowserBehavior
+  )
+}
+```
+
+#### Construction and Starting Video Processing
+
+The construction of the `DefaultVideoTransformDevice` will not start the camera or start processing. The method `meetingSession.audioVideo.chooseVideoInputDevice` is needed to be called. Device controller will use the inner `Device` to acquire the source `MediaStream` and start the processing pipeline at the same frame rate as the source `MediaStream`.
+The parameters to `chooseVideoInputQuality` will be respected when device controller acquires the source `MediaStream`. 
+After video input is chosen, `meetingSession.audioVideo.startLocalVideoTile` can be called to start streaming video.
+
+
+#### Switching Device on VideoTransformDevice
```

Comment:
Maybe I'm wrong here but i feel like it needs '(the) (Inner) device' to be grammatically correct?